databases/week2/week2-homework-part1.sql

```diff
@@ -0,0 +1,35 @@
+-- Part 1: Working with tasks
+
+
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (title, description, created, updated, due_date, status_id, user_id) values 
+('Learn about TypeScript', 'Study the basics and advanced features of TypeScript for better development practices', '2024-07-10 14:35:00', '2024-07-11 10:00:00', '2024-07-12 18:00:00', 3, 1);
+
+
+-- Change the title of a task
+UPDATE task
+SET title = 'Learn about TypeScript and NodeJS'
+WHERE task.id = LAST_INSERT_ID ();
```

Comment:
Nice thing experimenting with Last_indsert_id. However, in a real application, I wouldn't use it since most applications are concurrent (multiple users might be inserting data simultaneously), which means another user might insert something in the db while I was trying to do the update. This would change the title to a different task and not the one that I intended.